From c30bb2d0bda6d10c09a2959d1172ed43bfe5e321 Mon Sep 17 00:00:00 2001
From: Koji Matsuoka <koji.matsuoka.xm@renesas.com>
Date: Wed, 31 Jul 2013 17:09:49 +0900
Subject: [PATCH 1061/1083] drm: rcar-du: Fix RGB output port for r8a7790

DU0 is for HDMI only if adv7511 is enabled in r8a7790.

Signed-off-by: Koji Matsuoka <koji.matsuoka.xm@renesas.com>
---
 drivers/gpu/drm/rcar-du/rcar_du_drv.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/gpu/drm/rcar-du/rcar_du_drv.c b/drivers/gpu/drm/rcar-du/rcar_du_drv.c
index 1c1e9e0..673c3c5 100644
--- a/drivers/gpu/drm/rcar-du/rcar_du_drv.c
+++ b/drivers/gpu/drm/rcar-du/rcar_du_drv.c
@@ -253,7 +253,11 @@ static const struct rcar_du_device_info rcar_du_r8a7790_info = {
 		 * (currently unsupported) TCON output.
 		 */
 		[RCAR_DU_OUTPUT_DPAD0] = {
+#if defined(CONFIG_DRM_ADV7511)
+			.possible_crtcs = BIT(2) | BIT(1),
+#else
 			.possible_crtcs = BIT(2) | BIT(1) | BIT(0),
+#endif
 			.encoder_type = DRM_MODE_ENCODER_NONE,
 		},
 		[RCAR_DU_OUTPUT_LVDS0] = {
-- 
1.8.3.2

